package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "coverage": "nyc mocha",
     "start": "node index start",
-    "start-dev": "npm-run-all --parallel watch start",
+    "start-dev": "concurrently \"npm run start\" \"npm run watch\" --names \"server,webpack\" --prefix-colors \"yellow,blue\"",
```

Comment:
I'm not sure we need this. All TL outputs already start with `<timestamp> [<log level>]` so in a way we know what's TL and what's Webpack.
Not opposed to it though, but we would need a few changes:

- It currently removes all colors/styles in our CLI outputs so things like `2018-01-07 04:11:35 [INFO] Available at http://0.0.0.0:9000/ in private mode` appear in all white (muting important information). I haven't looked too much into `concurrently`, but I wouldn't be surprised if there was a way to pass it a flag to force colors or something

- `concurrently` and `npm-run-all` kind of duplicate each other as they solve similar issues. It would be better if we could use one or the other. `npm-run-all` has the `--print-label` flag that acts similarly to the colored prefixes of `concurrently`, except that:
  - https://github.com/mysticatea/npm-run-all/pull/115 has not been released yet, and since `"start".length === "watch".length`, both have the same color, lol!
  - Same issue as concurrently for colors other than the labels. That can be solved by adding `--colors` to `webpack --watch`, but for us, we need a new `--colors` option that [forces `colors.enabled = true;`](https://github.com/Marak/colors.js/issues/121#issuecomment-155014295) on non-TTY or something so this isn't free.

Reply:
Interesting, authors of all these packages have been saying [they want to all stop theirs and contribute to a single `npm-run-all` package](https://github.com/mysticatea/npm-run-all/issues/10). It doesn't seem like there is much efforts currently, but https://github.com/mysticatea/npm-run-all/issues/10#issuecomment-338668498.

Reply:
> but for us, we need a new --colors option that forces colors.enabled = true; on non-TTY or something so this isn't free.

Done in #1948 if we want this.